client/a8c-for-agencies/components/pressable-usage-details/index.tsx

```diff
@@ -115,6 +138,72 @@ export default function PressableUsageDetails( { existingPlan }: Props ) {
 					</div>
 				</div>
 			</div>
+			{ shouldRenderTitanCard && (
+				<div className="pressable-usage-details__info">
+					<div className="pressable-usage-details__info-item">
+						<div className="pressable-usage-details__info-header">
+							<div className="pressable-usage-details__info-label">
+								{ translate( 'Titan Email' ) }{ ' ' }
+								<span className="pressable-usage-details__addon-tag">
+									{ translate( 'ADD-ON' ) }
+								</span>
+							</div>
+							<div className="pressable-titan-usage-details__info-top-right">
+								{ translate( '%(total_inboxes)s inboxes', {
+									args: {
+										total_inboxes: totalActiveInboxes,
+									},
+									comment: '%(total_inboxes)s is the total number of active email inboxes.',
+								} ) }
+								<div className="pressable-titan-usage-details__info-inbox-price">
+									$3.5 per inbox monthly
+								</div>
+							</div>
+						</div>
+						<hr />
+						<div className="pressable-usage-details__info-value">
+							<div className="pressable-usage-details__titan-domains">
+								{ activeOrders.map( ( order: TitanOrder ) => (
+									<div key={ order.domain } className="pressable-usage-details__titan-domain-item">
+										<div className="pressable-usage-details__titan-domain-info">
+											<span className="pressable-usage-details__titan-domain">
+												{ order.domain }
+											</span>
+											<span className="pressable-usage-details__titan-plan-label">
+												{ translate( 'standard' ) }
+											</span>
+											{ order.order_plan === 'trial' && order.trial_end_at && (
+												<>
+													<span className="pressable-usage-details__titan-trial-badge">
+														{ translate( 'TRIAL' ) }
```

Comment:
We don't need to allcaps this translation string but instead handle this via CSS.

```
.pressable-usage-details__titan-trial-badge {
   text-transform: upper-case;
}
```